--- /tmp/0ad/HWDetect.cpp.orig	2026-09-18 00:10:12.136467609 +0200
+++ source/ps/GameSetup/HWDetect.cpp	2026-09-18 00:10:12.148560519 +0200
@@ -23,7 +23,9 @@
 #include "lib/debug.h"
 #include "lib/external_libraries/curl.h"
 #include "lib/external_libraries/enet.h"
+#if CONFIG2_LOBBY
 #include "lib/external_libraries/gloox.h"
+#endif
 #include "lib/external_libraries/png.h"
 #include "lib/external_libraries/libsdl.h"
 #include "lib/path.h"
@@ -209,7 +211,9 @@
 	appendLibrary(LibraryReporter{rq, "boost"}.Add("version", BOOST_VERSION).MakeReport());
 	appendLibrary(LibraryReporter{rq, "enet"}.Add("version", std::to_string(ENET_VERSION)).MakeReport());
 	appendLibrary(LibraryReporter{rq, "fmt"}.Add("version", FMT_VERSION).MakeReport());
+#if CONFIG2_LOBBY
 	appendLibrary(LibraryReporter{rq, "gloox"}.Add("version", gloox_version()).MakeReport());
+#endif
 	appendLibrary(LibraryReporter{rq, "libicu"}.Add("version", U_ICU_VERSION).MakeReport());
 	appendLibrary(LibraryReporter{rq, "libcurl"}.Add("version", std::string(curl_version())).MakeReport());
 #if CONFIG2_AUDIO
